--- setup.py.bak	2010-11-03 16:06:08.606119836 -0400
+++ setup.py	2010-11-03 16:07:03.948370010 -0400
@@ -43,7 +43,7 @@
     url="http://pythonpaste.org/deploy/",
     license='MIT',
     namespace_packages=['paste'],
-    packages=find_packages(exclude='tests'),
+    packages=find_packages(exclude=['tests']),
     package_data=find_package_data(
         exclude_directories=standard_exclude_directories + ('tests',)),
     zip_safe=False,
